src/websockets/controllers.md

```diff
@@ -0,0 +1,114 @@
+# Controllers
+
+Most of the concepts discussed elsewhere in this documentation, such as dependency injection, decorators, exception filters, pipes, guards and interceptors, apply equally to controllers. Wherever possible, Danet abstracts implementation details so that the same components can run across HTTP-based platforms and WebSockets. This section covers the aspects of Danet that are specific to WebSockets.
+
+In Danet, a websocket contrller is simply a class annotated with `@WebSocketController()` decorator. Danet does not use any third party library to handle websocket, we use the [WebApi WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API)
+
+
+## Overview
+
+In general, each WebSocketController is listening on the same port as the **HTTP server** and accessible on the given endpoints. For example, if you want to connect on `ws://youwebsite.com/ws` do the following: 
+
+```typescript
+@WebSocketController('ws')
+```
+
+The controller is now listening, but we have not yet subscribed to any incoming messages. Let's create a handler that will subscribe to the `events` messages and respond to the user with the exact same data.
+
+```ts events.controller.ts
+@SubscribeMessage('events')
+handleEvent(@Body() data: string): string {
```

Comment:
let's remove the `string` type, as it's different from the actual return of the example